src/common/common.go

```diff
@@ -132,3 +132,36 @@ func IsK8sNodeIsReady(node v1.Node) bool {
 	}
 	return false
 }
+
+// BuildHostsMapIPAddressBased builds a map containing all the IP addresses of the nodes in the
```

Comment:
```suggestion
// BuildHostsMapIPAddressBased builds a map containing all the IP addresses of the hosts in the
```

I normally use hosts to refer to hardware that can, potentially, become a node in the cluster. It sounds silly, but it's just a way to separate concerns and stages and provide some implicit context when reasoning about problems. For example, a host may or may not be part of a cluster, whereas a node is considered to have been registered in the cluster.

Reply:
Done